pom.xml

```diff
@@ -86,7 +86,7 @@
         <postgresql.version>42.2.5</postgresql.version>
         <hive.jdbc.version>2.1.0</hive.jdbc.version>
         <commons.io.version>2.4</commons.io.version>
-        <oshi.core.version>3.9.1</oshi.core.version>
+        <oshi.core.version>6.1.1</oshi.core.version>
```

Comment:
This version of 'osh-core' is changed, at the same time  the license need to be checked in './dolphinscheduler-dist/release-docs/' .